app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -206,23 +206,28 @@ def remove_subscriptions
 
     api :PUT, "/activation_keys/:id/content_override", N_("Override content for activation_key")
     param :id, :identifier, :desc => N_("ID of the activation key"), :required => true
-    param :label, String, :desc => N_("Label of the content"), :required => false
-    param :value, :number, :desc => N_("Override value 0/1"), :required => false
+    param :content_override, Hash, :desc => N_("Content override parameters") do
+      param :content_label, String, :desc => N_("Label of the content"), :required => true
+      param :value, :bool, :desc => N_("Override to 0/1, or 'default'"), :required => true
+    end
```

Comment:
I actually didn't realize hammer could handle nested params. I've tested it in hammer and it seems to work, and removing the different handling of the two ways to handle cleans the code up a bit.